idaes/models/unit_models/product.py

```diff
@@ -26,7 +26,10 @@
 
 # Product blocks can reuse the Feed initializer
 # For consistency and future proofing, import with new name
-from idaes.models.unit_models.feed import FeedInitializer as ProductInitializer
+from idaes.models.unit_models.feed import (
+    FeedInitializer as ProductInitializer,
+    FeedScaler as ProductScaler,
+)
```

Comment:
Is there a better name we can give the FeedInitializer and FeedScaler since they'll also be used for products and statejunctions?